src/test/CMakeLists.txt

```diff
@@ -41,11 +42,11 @@ set(
 include_directories(${CMAKE_CURRENT_SOURCE_DIR})
 
 # Configure hyriseTest
-add_executable(hyriseTest ${HYRISE_TEST_SOURCES})
+add_executable(hyriseTest ${HYRISE_TEST_SOURCES} storage/fixed_width_integer_vector_test.cpp)
```

Comment:
Not sure, why you added this explicitly against if the cpp in the sources list above.